mohanadarafe/GuideMe
# feat: add logo to splash screen (#120)

Added logo to splash screen when the app loads.
fixes #120 

**To test:** 

1. Start the app
`expo start`

2. Check the loading screen :)
